conans/client/loader_parse.py

```diff
@@ -29,7 +29,7 @@ class defining the Recipe, but also process possible existing generators
     """
     result = None
     for name, attr in conanfile_module.__dict__.items():
-        if "_" in name:
```

Comment:
Mostly. This would skip things like ``__builtins__
__file__
__package__
__name__
__doc__`` 
If not skipped, it is not a big problem, later checks are looking for something subclassing ``ConanFile``.